<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchStudy</class>
 <widget class="QWidget" name="SwitchStudy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Ausrichtung</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="leftHandedButton">
          <property name="text">
           <string>Linksweiche</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rightHandedButton">
          <property name="text">
           <string>Rechtsweiche</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Zustand</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="leftButton">
          <property name="text">
           <string>links</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rightButton">
          <property name="text">
           <string>rechts</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Z채hlrichtung</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="backwardButton">
          <property name="text">
           <string>r체ckw채rts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="forwardButton">
          <property name="text">
           <string>vorw채rts</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Abschnitt</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="freeButton">
          <property name="text">
           <string>frei</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="shuntButton">
          <property name="text">
           <string>Rangieren</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tourButton">
          <property name="text">
           <string>Fahrt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="occupiedButton">
          <property name="text">
           <string>belegt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="passedButton">
          <property name="text">
           <string>durchgefahren</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Verschluss</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QRadioButton" name="openButton">
          <property name="text">
           <string>offen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="transitButton">
          <property name="text">
           <string>Umlauf</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="closedButton">
          <property name="text">
           <string>verschlossen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="symbolWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="mrw::ui::RegularSwitchWidget" name="smallSwitchWidget" native="true">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="mrw::ui::RegularSwitchWidget" name="bigSwitchWidget" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mrw::ui::RegularSwitchWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/regularswitchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
